specification/components/schemas/codeable/RemovalReasonExitCode.yaml

```
@@ -23,13 +23,19 @@ properties:
           type: string
           description: A CodeSystem that identifies the reason a PDS record has been removed.
           example: SCT
-          enum:
-            - DEA
-            - EMB
-            - SCT
-            - NIT
-            - TRA
-            - ORR
+          oneOf: 
+             const: DEA
+              description: Death
+              const: EMB
+              description:  Embarkation
+              const: SCT
+              description: Transferred to Scotland
+              const: NIT
+              description: Transferred to Northern Ireland
+              const: TRA
+              description: Temporary resident not returned
+              const: ORR
+              description: Other reason
```

I think this will be the required syntax
```suggestion
          oneOf: 
            - const: DEA
              description: Death
            - const: EMB
              description:  Embarkation
            - const: SCT
              description: Transferred to Scotland
            - const: NIT
              description: Transferred to Northern Ireland
            - const: TRA
              description: Temporary resident not returned
            - const: ORR
              description: Other reason
```

---

Ah thanks 👍